example_docs/docs/conf.py

```diff
@@ -10,58 +10,22 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-# pylint: disable=invalid-name
-# Configuration file for the Sphinx documentation builder.
-#
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# http://www.sphinx-doc.org/en/master/config
-
-# -- Path setup --------------------------------------------------------------
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
-
-"""
-Sphinx documentation builder
-"""
 import os
 import sys
-sys.path.insert(0, os.path.abspath('.'))
-# -- Project information -----------------------------------------------------
-project = 'Qiskit sphinx theme'
-copyright = '2020, Qiskit Development Team'  # pylint: disable=redefined-builtin
-author = 'Qiskit Development Team'
 
-import qiskit_sphinx_theme
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Using `..` rather than `.` because this makes https://github.com/Qiskit/qiskit_sphinx_theme/pull/265 simpler.